### ISSUE TITLE
output kv log lag

### ISSUE THREAD

**Issue by Gonioul** [pbkhrv/rtl_433-hass-addons#85]
output kv seems to be buffered, should be unbuffered to have a live debug.

**Comment by korylprince** [comment]
Yeah, I've hit this same issue and was going a little crazy over it...

@deviantintegral This is because sed is buffered by default (unless connected to a terminal). The default sed in the Docker image is busybox's sed, which doesn't have a `-u` flag. So the Dockerfile would need `sed` added to it's `apk add` list, then the `-u` option should be used for anywhere sed is used in `/run.sh`.
